### PR TITLE
Add Javascript First Solution :)

### PR DESCRIPTION
**Pull request by vinasxz**

### CODE REVIEW

File: src/calculateRentalCost.js [hunk 1]
```diff
@@ -4,7 +4,15 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  let totalCost = days * 40;
+
+  if (days >= 7) {
```
Comment (jv-aquino) [inline src/calculateRentalCost.js:9]: The conditional days should be stored in a way to facilitate changes - also remember the naming guidelines (check 1) when you do this
☝️
[checklist 2](https://github.com/mate-academy/js_task-transportation-on-vacation/blob/master/checklist.md)

File: src/calculateRentalCost.js [hunk 2]
```diff
@@ -4,7 +4,15 @@
...
+    totalCost -= 50;
```
Comment (jv-aquino) [inline src/calculateRentalCost.js:10]: you should also store the discount in a better manner

File: src/calculateRentalCost.js [hunk 3]
```diff
@@ -4,7 +4,15 @@
...
+  } else if (days >= 3) {
+    totalCost -= 20;
```
Comment (jv-aquino) [inline src/calculateRentalCost.js:12]: Also, you **could/might** use a direct return after the conditionals

[and remember checklist 4 and 5](https://github.com/mate-academy/js_task-transportation-on-vacation/blob/master/checklist.md)

**Review by jv-aquino — CHANGES_REQUESTED** [review]
Good code but we can make it better!

File: src/calculateRentalCost.js (hunk 3 above)
Comment (vinasxz) [reply src/calculateRentalCost.js:12]: Wow, i'm guessing i dind't saw that. Fixed already, following the checklist now.

**Pull request closed without merging**